src/reads-stdin.c

```diff
@@ -0,0 +1,68 @@
+#include <stdio.h>
+#include <string.h>
+
+// This program reads stdin and asserts that it is the expected value repeated
+// until stdin ends
+
+// Intentionally a prime length to make hitting buffering edge cases more likely
+const char expected_repeat[] = {'e', 'x', 'a', 'm', 'p', 'l', 'e'};
```

Comment:
Either way is fine with me, but wanted to suggest:
```suggestion
const char expected_repeat[7] = "example";
```

Reply:
Given that the content doesn't matter I prefer separated characters over a magic number.

Reply:
Your suggestion deleted this variable.